server/manager/src/main/java/org/apache/accumulo/manager/metrics/fate/meta/MetaFateMetrics.java

```diff
@@ -59,7 +60,8 @@ public void registerMetrics(MeterRegistry registry) {
   @Override
   protected ReadOnlyFateStore<FateMetrics<MetaFateMetricValues>> buildStore(ServerContext context) {
     try {
-      return new MetaFateStore<>(getFateRootPath(context), context.getZooReaderWriter());
+      return new MetaFateStore<>(getFateRootPath(context), context.getZooReaderWriter(),
+          AbstractFateStore.createDummyLockID(), null);
```

Comment:
This store is being created for metrics, so it may be using it in a readonly way and may never reserve. If it never reserves then it does not need a lock, not even a dummy lock.  Not something for this PR, but for this case I think we want to somehow create a store obj where attempts to reserve will be non functional and throw an exception.

Reply:
Yeah, can leave this unresolved for making a follow on issue. I put the dummy lock in many places where a store was created, without regard to whether reservations would actually occur. A follow on issue can address this.